source/api/presentation/3.0/index.md

```diff
@@ -1298,81 +1292,117 @@ Ranges _MAY_ link to an Annotation Collection that has the content of the Range
+These Annotations are collected together in Annotation Page resources, which are included in the `items` property from the Canvas. Each Annotation Page can be embedded in its entirety, if the Annotations should be processed as soon as possible when the user navigates to that Canvas, or a reference to an external page. This reference _MUST_ include `id` and `type`, _MUST NOT_ include `items` and _MAY_ include other properties, such as `behavior`. All of the Annotations in the Annotation Page _SHOULD_ have the Canvas as their `target`. Embedded Annotation Pages _SHOULD_ be processed by the client first, before externally referenced pages.
+Clients _SHOULD_ process the Annotation Pages and their items in the order given in the Canvas. 
+
+An Annotation Page _MUST_ have an HTTP(S) URI given in `id`, and _MAY_ have any of the other properties defined in this specification or the Web Annotation specification. The Annotations are listed in the `items` property of the Annotation Page.
 
 ``` json-doc
 {
   "@context": [
     "http://www.w3.org/ns/anno.jsonld",
     "http://iiif.io/api/presentation/{{ page.major }}/context.json"
   ],
-  "id": "https://example.org/iiif/book1/annocoll/transcription",
-  "type": "AnnotationCollection",
-  "label": { "en": [ "Diplomatic Transcription" ] },
+  "id": "https://example.org/iiif/book1/annopage/p1",
+  "type": "AnnotationPage",
 
-  "first": { "id": "https://example.org/iiif/book1/annopage/l1", "type": "AnnotationPage" },
-  "last": { "id": "https://example.org/iiif/book1/annopage/l120", "type": "AnnotationPage" }
+  "items": [
+    {
+      "id": "https://example.org/iiif/book1/annopage/p1/a1",
+      "type": "Annotation"
+      // ...
+    },
+    {
+      "id": "https://example.org/iiif/book1/annopage/p1/a2",
+      "type": "Annotation"
+      // ...
+    }
+  ]
 }
 ```
 
-### 5.7. Collection
+### 5.6. Annotations
+
+Annotations follow the [Web Annotation][webanno] data model. The description provided here is a summary plus any IIIF specific requirements. It must be noted that the W3C standard is the official documentation.
 
-Collections are used to list the Manifests available for viewing, and to describe the structures, hierarchies or sets that the resources are part of. Collections _MAY_ include both other Collections and Manifests, in order to form a tree-structured hierarchy. Collections might be used to model dynamic result sets from a search, fixed sets of related resources, or other groupings of Manifests for presentation to the user, typically for navigation amongst the member items.
+Note that the Web Annotation data model defines different patterns for the `value` property, when used within an Annotation. The value of a Textual Body or a Fragment Selector, for example, are strings rather than JSON objects with languages and values. Care _MUST_ be taken to use the correct string form in these cases.
```

Comment:
I don't think it make sense to use RFC _MUST_ when talking about care, suggest changing to plain must

Reply:
:+1: Fixed.